midi_app_controller/gui/binds_editor.py

```diff
@@ -170,12 +192,33 @@ def _create_button_layout(self, button_id: int, button_name: str) -> QGridLayout
     def get_binds(self) -> list[ButtonBind]:
         """Returns list of all binds currently set in this widget."""
         result = []
-        for button_id, combo in self.button_combos:
+        for button_id, combo in self.button_combos.items():
             action = combo.get_selected_action_id()
             if action is not None:
                 result.append(ButtonBind(button_id=button_id, action_id=action))
         return result
 
+    @ensure_main_thread(
+        await_return=True
+    )  # QTimer can only be used in the main thread.
+    def highlight_button(self, button_id: int) -> None:
+        """Highlights combos associated with `button_id`.
+
+        Starts a timer that unhighlights the combos after `HIGHLIGHT_DURATION_MS`.
+        """
+        if self.stop:
+            return
+
+        combo = self.button_combos[button_id]
+        combo.setStyleSheet(HIGHLIGHT_STYLE_SHEET)
+
+        self.highlight_timers[button_id].start(HIGHLIGHT_DURATION_MS)
+
+    def stop_highlighting_button(self, button_id: int) -> None:
+        """Unhighlights combos associated with `knob_id`."""
+        combo = self.button_combos[button_id]
+        combo.setStyleSheet("")
+
 
 class KnobBinds(QWidget):
```

Comment:
Not necessarily in this branch, but I think you should create a superclass for ButtonBinds and KnobBinds - some of the code is repeated